pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -629,6 +629,8 @@ class Build {
         }
         def weekly = ''
         if ( Boolean.valueOf(buildConfig.WEEKLY) ) { weekly = '_weekly'}
+        jckRerunSummary.appendText('<b>RERUN JCK TESTS:</b><ul>', false)
+        def aqa_test_pipeline_BaseURL = "https://ci.adoptium.net/view/Test_grinder/job/AQA_Test_Pipeline/parambuild?"
```

Comment:
Note for myself as I can't login my local workspace-  ? is an extra needs to be removed.